tools/json_tools/generic_guns_validator.py

```diff
@@ -25,7 +25,7 @@
     'chemical_spray',
     'fishspear',
     'flammable',
-    'gene_sting_barb',
+    'gene_sting',
```

Comment:
How is this change not breaking generic guns?  And if this is correct how did the previous PR not break GG?

Reply:
This list contain only ammo types, not ammo ids
About why it didn't break the PR before it, probably because it used custom damage type, not registered by the script?